lib/Service/ColumnTypes/SelectionBusiness.php

```diff
@@ -18,7 +18,7 @@ public function parseValue($value, ?Column $column = null): string {
 		}
 
 		$intValue = (int) $value;
-		if ((string)$intValue === $value) {
+		if ((string) $intValue === (string) $value) {
```

Comment:
```suggestion
		if ((string)(int)$value === (string)$value) {
```

I guess these two lines can be unified into one line.

Reply:
Ah, if you need `$intValue` later on, nevermind.